app/uk/gov/hmrc/cgtpropertydisposals/models/des/returns/DesDisposalType.scala

```diff
@@ -27,9 +27,13 @@ object DesDisposalType {
 
```

Comment:
remove this `final` modifiers - not necessary

Reply:
done